### ISSUE TITLE
redo move encoding

### ISSUE THREAD

**Issue by a-teammate** [inexor-game/code#347]
Currently we just drop almost uncompressed AVI files using the internal function to encode movies.

I suggest we use the webm-SDK to compress them (http://www.webmproject.org/docs/webm-sdk/example_vpxenc.html). Reading their docs this can even be done in some background thread, changing just some parameters.

I think this topic can be improved relatively separately from the other code refractorings so it is a good-first-bug. However you must be able to read the C documentation of libvpx (its inside the webm-SDK).  
Basically what we need to do is: having some input RGB frames. Encode them using the SDK and save them to a file.

If someone signals to start working on this, ping me and I'll provide the necessary conan.io-package so you just need to code, not worry about how to get the webm-SDK into the project.
